crates/cheatnet/src/forking/state.rs

```diff
@@ -165,9 +167,12 @@ impl StateReader for ForkStateReader {
                     .cache_get_class_hash_at(contract_address, class_hash);
                 Ok(class_hash)
             }
+            Err(ProviderError::StarknetError(StarknetError::ContractNotFound)) => {
+                Ok(Default::default())
+            }
```

Comment:
Is this okay to do?

Reply:
Yes - we have to do it because it's what blockifier expects (during i.e. deploy, we check for this and if the hash is default, we allow deployment)